en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,55 @@
 # Integration Concepts
-Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
- 
+Choreo integrations allow you to create enterprise integrations to automate workflows in a user-friendly low-code environment. Here are some basic concepts which will help you understand Choreo integrations as you develop your integrations in the [Choreo Console]({{choreo_console}}).
```

Comment:
Shall we say "Choreo allows you to . .  " instead of "Choreo integrations allow you to"?

Shall we say "Here are some basic concepts to help you understand how to build integrations using the Choreo"?

Reply:
+1 .
-1, This does not exactly help you understand how to build, rather helps you understand how Integrations work in Choreo.